packages/client/katello-host-tools/katello-host-tools.spec

```diff
@@ -328,25 +236,13 @@ exit 0
 %{python_libdir}/katello_host_tools-*.egg-info
 %exclude %{katello_libdir}/contrib
 
-%if %{build_agent}
-%exclude %{katello_libdir}/agent
-%endif
-
 %if %{build_tracer}
-# TODO pycached unavailable on SuSE - what to do with cache files?
-%if %{zypper_install}
-%exclude %{katello_libdir}/tracer.py
-%else
-%if %{yum_install}
-%exclude %{katello_libdir}/tracer.py*
-%else
-%pycached %exclude %{katello_libdir}/tracer.py
-%endif
-%endif
-
 %if %{dnf_install}
+%pycached %exclude %{katello_libdir}/tracer/*.py
+%exclude %{katello_libdir}/tracer/__pycache__
```

Comment:
I wonder if moving this block to the tracer subpackage (and dropping the `%exclude`) will fix the "el8/9 has empty tracer dir" complaint I had